src/plugins/intel_cpu/src/nodes/scaled_attn.cpp

```diff
@@ -1238,10 +1238,17 @@ void ScaledDotProductAttention::createPrimitive() {
         std::shared_ptr<Executor> executor = nullptr;
 #ifdef OPENVINO_ARCH_X86_64
         if (rtPrecision == ov::element::bf16) {
-            executor = std::make_shared<AttentionExecutor<KT_ONEDNN, ov::bfloat16>>(context,
-                                                                                    m_key_quant_param.groupSize,
-                                                                                    m_value_quant_param.groupSize,
-                                                                                    m_key_quant_param.isByChannel);
+            if (ov::with_cpu_x86_bfloat16() && !ov::with_cpu_x86_avx2_vnni_2()) {
```

Comment:
I would suggest to fallback on FP32 instead of BF16 with reference. I should be much more performant option

Reply:
based on our teams sync, will keep this changes to align with f16 implement.